SU-HSDO/suhumsci
# ARCH-71 Set up publications display and masonry pattern

# READY FOR REVIEW

# Summary
- Set the publications node display using layout builder
- Added the publications view using masonry pattern.

# Needed By (Date)
- next week

# Urgency
- Low

# Steps to Test

1. Checkout this branch
2. install a clean version
2a. suggested disable `acquia_connector`, `simplesamlphp_auth`, & `config_readonly`
3. enable hs_publications
4. create some publications via the UI or devel_generate
5. ensure a publication page looks somewhat normal
6. create an empty "Publications" basic page.
7. click on "layout" to change the basic page contents
8. add the publications masonry list view and the exposed filter block
9. ensure masonry looks correct and the view functions as expected. [mockup](https://projects.invisionapp.com/d/main#/console/14029316/292719604/preview)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
